tests/e2e/cli/test_cli.py

```diff
@@ -294,10 +294,10 @@ def test_otx_explain_e2e_cli(
         "explain",
         "--config",
         recipe,
-        "--model.num_classes",
+        "--model.label_info",
         "1000",
```

Comment:
Could you explain about this change? I cannot understand mean of the comment

Reply:
I think it needs to be updated after #3344

Reply:
It was my attempt to fix broken e2e after https://github.com/openvinotoolkit/training_extensions/pull/3344, I'll revert it